cmd/execute-cadence-script/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/flow-go-sdk/client"
+
+	"github.com/rs/zerolog"
+	"github.com/spf13/pflag"
+	"google.golang.org/grpc"
+)
+
+func main() {
+
+	var (
+		flagAPI        string
+		flagScriptFile string
+		flagLogLevel   string
+		flagHeight     int64
+	)
+
+	// TODO: mismatch between block height type - using int64 instead of uint64
+
+	pflag.StringVarP(&flagScriptFile, "script-file", "s", "", "cadence script to execute")
+	pflag.StringVarP(&flagAPI, "api", "a", "localhost:3569", "access node API address")
+	pflag.StringVarP(&flagLogLevel, "log-level", "l", "info", "log level for JSON logger")
+	pflag.Int64VarP(&flagHeight, "height", "h", -1, "height on which to execute script")
```

Comment:
I started introducing shorter command line flags, as separating the binaries made them less ambiguous. I recommend to simply`script` and `log` here and for the flag variable names.